lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -28,23 +29,26 @@
     get_milestone_relationship_types,
     seed_milestone_relationship_types,
 )
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+
 from student.models import CourseEnrollment
 from student.tests.factories import CourseEnrollmentFactory, AnonymousUserFactory
-from mock import patch, Mock
-import mock
 
 
+@patch.dict('django.conf.settings.FEATURES', {'MILESTONES_APP': True, 'ENTRANCE_EXAMS': True})
 class EntranceExamTestCases(LoginEnrollmentTestCase, ModuleStoreTestCase):
     """
-    Check that content is properly gated.  Create a test course from scratch to mess with.
-    We typically assume that the Entrance Exam feature flag is set to True in test.py
-    However, the tests below are designed to execute workflows regardless of the setting
-    If set to False, we are essentially confirming that the workflows do not cause exceptions
+    Check that content is properly gated.
+
+    Creates a test course from scratch. The tests below are designed to execute
+    workflows regardless of the feature flag settings. If set to False, we are
+    essentially confirming that the workflows do not cause exceptions
     """
+    @patch.dict('django.conf.settings.FEATURES', {'ENTRANCE_EXAMS': True, 'MILESTONES_APP': True})
     def setUp(self):
-        """
-        Test case scaffolding
-        """
+        """ Test case scaffolding """
```

Comment:
No reason for this change.
In fact, it's counter to edX's python guidelines for docStrings: https://github.com/edx/edx-platform/wiki/Python-Guidelines